index.html

```diff
@@ -191,18 +237,28 @@ <h2 class="aboutme-card-title">Skills</h2>
     <!--end about me section-->
 
       <!--start of footer section / contact form-->
-      <footer>
-      <form  method="post" action="https://formspree.io/f/xyyowwnj">
-        <section class="contact-form-container">
+    <section class="contact-form-container"> 
         <h2 class="contact-form-title">
-                I&#39;m always interested in hearing about new projects, so if you&#39;d like to chat please get in touch.</h2>
```

Comment:
- [ ] Please make contact form responsive for desktop too. Please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Here you have a screenshot of contact form:

![image](https://user-images.githubusercontent.com/74506933/154660808-57d11a91-d274-4acd-ad14-16e8a6f5a655.png)